services/API-service/src/api/event/event.service.ts

```diff
@@ -162,15 +161,27 @@ export class EventService {
         event.disasterSpecificProperties = {};
       }
       event.disasterSpecificProperties.eapAlertClass =
-        await this.getEventEapAlertClass(
-          disasterSettings,
-          event.forecastSeverity,
-          event.forecastTrigger,
-        );
+        await this.getEventEapAlertClass(disasterSettings, event.alertLevel);
     }
     return rawEvents;
   }
 
+  public getAlertLevel(event: EventSummaryCountry): AlertLevel {
+    if (event.userTrigger || event.forecastTrigger) {
+      return AlertLevel.TRIGGER;
+    }
+
+    if (event.forecastSeverity > 0.7) {
+      return AlertLevel.WARNING;
+    } else if (event.forecastSeverity > 0.3) {
+      return AlertLevel.WARNINGMEDIUM;
+    } else if (event.forecastSeverity > 0) {
+      return AlertLevel.WARNINGLOW;
+    }
```

Comment:
This works for now, but will need updating as we go along (which is fine). I'm not particularly happy with this definition of WARNING being placed "above" low/medium warning. But again, it works for now, so good enough. We will already touch upon this when we would facilitate high-warning in floods.

Reply:
I don't like it either but I followed the definition from [here](https://github.com/rodekruis/IBF-system/blob/fa0266bc9d645d0c35cf15cd75bcfe16233884dd/services/API-service/src/api/event/event.service.ts#L495-L500).

I prefer TRIGGER > HIGH WARNING > WARNING > LOW WARNING > NONE

Reply:
let's leave it for now, and just touch upon it again in the High Warning item (or whenever it comes up more pressing).